modules/tinymce/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Templates will be parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS. #TINY-9244
 - Pressing backspace in an empty line now preserves formatting in a previous empty line. #TINY-9454
 - Pressing enter inside the `inputfontsize` input would not move the focus back into the editor content. #TINY-9598
+- Help dialog horizontal rule now have a higher contrast in dark mode. #Tiny-9587
```

Comment:
```suggestion
- Improved contrast of table borders within dialogs in the `oxide-dark` skin. #TINY-9587
```
This also affects the wordcount dialog, but should be more specific to the location of the change.